clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -417,7 +420,7 @@ private boolean trySend(long now) {
 
     public void maybeTriggerWakeup() {
         if (!wakeupDisabled.get() && wakeup.get()) {
-            log.trace("Raising wakeup exception in response to user wakeup");
+            log.debug("Raising WakeupException in response to user wakeup");
```

Comment:
Funnily enough, this was actually the only thing I set out to change initially (notice the branch name), but I wanted to ensure that the message included the consumer context. I will update the description.